internal/api/health_check.go

```diff
@@ -20,15 +20,18 @@ type healthCheckResponse struct {
 	Description string                       `json:"description"`
 }
 
+// initHealthCheck health check endpoints on the given router.
```

Comment:
Small GoDoc-tweak, to create a sentence:
```suggestion
// initHealthCheck initializes health check endpoints on the given router.
```